app/src/sharedTest/java/org/oppia/app/walkthrough/WalkthroughActivityTest.kt

```diff
@@ -41,6 +41,17 @@ class WalkthroughActivityTest {
     }
   }
 
+
+  @Test
+  fun testWalkthroughFragment_increaseProgress_configurationChanged_worksCorrectly() {
```

Comment:
Add back button test case also, meaning:

1. Go to second fragment
2. Orientation change
3. Click on back button
4. Check the content.